cluster/centos/util.sh

```diff
@@ -316,10 +333,34 @@ function provision-node() {
 #   KUBE_TEMP
 function ensure-setup-dir() {
   kube-ssh "${1}" "mkdir -p ${KUBE_TEMP}; \
+                   mkdir -p ${KUBE_TEMP}/etcd-cert; \
                    sudo mkdir -p /opt/kubernetes/bin; \
                    sudo mkdir -p /opt/kubernetes/cfg"
 }
 
+# Generate certificates for etcd cluster
+#
+# Assumed vars:
+#   $1 (etcd member name)
+#   $2 (master ip)
+function ensure-etcd-cert() {
+  local etcd_name="$1"
+  local master_ip="$2"
+  local cert_dir="${ROOT}/etcd-cert"
+
+  mkdir -p "${cert_dir}"
```

Comment:
Isn't better to create `${cert_dir}` inside `generate-etcd-cert` function (if the dir does not exist). Currently, `generate-etcd-cert` function has undocumented assumption that the directory already exists.